tokio/src/process/mod.rs

```diff
@@ -690,6 +690,38 @@ impl Command {
         self
     }
 
+    /// Sets the process group ID (PGID) of the child process. Equivalent to a
+    /// setpgid call in the child process, but may be more efficient.
+    ///
+    /// Process groups determine which processes receive signals.
+    ///
+    /// **Note**: This is an [unstable API][unstable] but will be stabilised once
+    /// tokio's MSRV is sufficiently new. See [the documentation on
+    /// unstable features][unstable] for details about using unstable features.
+    ///
+    /// If you want similar behaviour without using this unstable feature you can
+    /// create a [`std::process::Command`] and convert that into a
+    /// [`crate::process::Command`] using the `From` trait.
```

Comment:
We don't use `crate` in our documentation.
```suggestion
    /// [`tokio::process::Command`] using the `From` trait.
```
If this link doesn't work out of the box, then you can add
```Rust
[`tokio::process::Command`]: crate::process::Command
```

Reply:
Ah, thanks for the pointer about the alias - I tried `tokio::process::Command` earlier and the docs wouldn't build. With that alias, it works. :)